pkg/utils/build/controller.go

```diff
@@ -0,0 +1,40 @@
+package build
+
+const (
+	BuildRequestAnnotationName                    = "build.appstudio.openshift.io/request"
+	BuildRequestTriggerSimpleBuildAnnotationValue = "trigger-simple-build"
+	BuildRequestConfigurePaCAnnotationValue       = "configure-pac"
+	BuildRequestUnconfigurePaCAnnotationValue     = "unconfigure-pac"
+
+	BuildStatusAnnotationName = "build.appstudio.openshift.io/status"
+)
+
+type BuildStatus struct {
+	Simple *SimpleBuildStatus `json:"simple,omitempty"`
+	PaC    *PaCBuildStatus    `json:"pac,omitempty"`
+	// Shows build methods agnostic messages, e.g. invalid build request.
+	Message string `json:"message,omitempty"`
+}
+
+// Describes persistent error for build request.
+type ErrorInfo struct {
+	ErrId      int    `json:"error-id,omitempty"`
+	ErrMessage string `json:"error-message,omitempty"`
+}
+
+type SimpleBuildStatus struct {
+	// BuildStartTime shows the time when last simple build was submited.
+	BuildStartTime string `json:"build-start-time,omitempty"`
+
+	ErrorInfo
+}
+
+type PaCBuildStatus struct {
+	// State shows if PaC is used.
+	// Values are: enabled, disabled.
+	State string `json:"state,omitempty"`
+	// Contains link to PaC provision / unprovision pull request
+	MergeUrl string `json:"merge-url,omitempty"`
+
+	ErrorInfo
+}
```

Comment:
can't we import these from [build-service](https://github.com/redhat-appstudio/build-service/pull/164/files#diff-55dabc319dc9f0a58a4c79d69d79368078f475199412ea72d3fae8bd7c659785) instead?